pkg/runtime/oci/config/config_linux.go

```diff
@@ -0,0 +1,9 @@
+package config
+
+const Platform = "linux"
+
+type RuntimeOciPlatform struct {
```

Comment:
I'm slightly confused how you're using the OCI runtime config. I see you have defined an interface, does the implementation of the interface methods happen in the OCI runtime spec package?

Reply:
No it just describe the whole interface with methods to implement in order to manipulate OCI specs